.github/workflows/run-tests.yml

```diff
@@ -240,24 +235,23 @@ jobs:
           CI: false
           DCP_DIAGNOSTICS_LOG_LEVEL: debug
           DCP_DIAGNOSTICS_LOG_FOLDER: ${{ github.workspace }}/testresults/dcp
+          # During restore and build, we use -ci, which causes NUGET_PACKAGES to point to a local cache (Arcade behavior).
+          # In this step, we are not using Arcade, but want to make sure that MSBuild is able to evaluate correctly.
+          # So, we manually set NUGET_PACKAGES
+          NUGET_PACKAGES: ${{ github.workspace }}/.packages
+        
         run: >
           ${{ env.DOTNET_SCRIPT }} test ${{ env.TEST_PROJECT_PATH }}
           /p:ContinuousIntegrationBuild=true
-          -s eng/testing/.runsettings
-          -l "console;verbosity=normal"
-          -l "trx;LogFilePrefix=${{ inputs.testShortName }}"
-          -l "GitHubActions;summary.includePassedTests=true;summary.includeSkippedTests=true"
-          "--blame"
-          --blame-hang-timeout ${{ inputs.testHangTimeout }}
-          --blame-crash
-          --results-directory testresults
+          -bl:${{ github.workspace }}/testresults/test.binlog
           --no-restore
           --no-build
-          -bl:${{ github.workspace }}/testresults/test.binlog
-          ${{ inputs.extraTestArgs }}
           --
-          RunConfiguration.CollectSourceInformation=true
-          RunConfiguration.TestSessionTimeout=${{ inputs.testSessionTimeoutMs }}
+          --report-trx --report-trx-filename "${{ inputs.testShortName }}.trx"
```

Comment:
did you notice whether hang/crash dumps go into the folder that is uploaded as part of test results? I can't find a doc on where they are written if you don't set filename.

Reply:
They should go into results directory (whatever specified as `--results-directory`). If `--results-directory` isn't specified, it should be defaulting to `TestResults` (relative to the test executable AFAIK, but I think you always specify `--results-directory` anyways)